diff -up libyuv-0/CMakeLists.txt.2~ libyuv-0/CMakeLists.txt
--- libyuv-0/CMakeLists.txt.2~	2023-01-04 00:25:10.080999909 +0100
+++ libyuv-0/CMakeLists.txt	2023-01-04 00:25:48.680277109 +0100
@@ -40,11 +40,11 @@ SET_TARGET_PROPERTIES	( ${ly_lib_shared}
 
 # this creates the conversion tool
 ADD_EXECUTABLE			( yuvconvert ${ly_base_dir}/util/yuvconvert.cc )
-TARGET_LINK_LIBRARIES	( yuvconvert ${ly_lib_static} )
+TARGET_LINK_LIBRARIES	( yuvconvert ${ly_lib_shared} )
 
 # this creates the yuvconstants tool
 ADD_EXECUTABLE      ( yuvconstants ${ly_base_dir}/util/yuvconstants.c )
-TARGET_LINK_LIBRARIES  ( yuvconstants ${ly_lib_static} )
+TARGET_LINK_LIBRARIES  ( yuvconstants ${ly_lib_shared} -lm )
 
 find_package ( JPEG )
 if (JPEG_FOUND)
@@ -70,7 +70,7 @@ if(TEST)
   endif()
 
   add_executable(libyuv_unittest ${ly_unittest_sources})
-  target_link_libraries(libyuv_unittest ${ly_lib_name} ${GTEST_LIBRARY})
+  target_link_libraries(libyuv_unittest ${ly_lib_shared} ${GTEST_LIBRARY})
   find_library(PTHREAD_LIBRARY pthread)
   if(NOT PTHREAD_LIBRARY STREQUAL "PTHREAD_LIBRARY-NOTFOUND")
     target_link_libraries(libyuv_unittest pthread)
