diff -up libyuv-0/CMakeLists.txt.4~ libyuv-0/CMakeLists.txt
--- libyuv-0/CMakeLists.txt.4~	2024-07-29 20:53:18.282655347 +0200
+++ libyuv-0/CMakeLists.txt	2024-07-29 20:53:56.335182863 +0200
@@ -155,8 +155,8 @@ ADD_EXECUTABLE      ( cpuid ${ly_base_di
 TARGET_LINK_LIBRARIES  ( cpuid ${ly_lib_shared} )
 
 # this creates the conversion tool
-ADD_EXECUTABLE			( yuvconvert ${ly_base_dir}/util/yuvconvert.cc )
-TARGET_LINK_LIBRARIES	( yuvconvert ${ly_lib_shared} )
+#ADD_EXECUTABLE			( yuvconvert ${ly_base_dir}/util/yuvconvert.cc )
+#TARGET_LINK_LIBRARIES	( yuvconvert ${ly_lib_shared} )
 
 # this creates the yuvconstants tool
 ADD_EXECUTABLE      ( yuvconstants ${ly_base_dir}/util/yuvconstants.c )
@@ -211,9 +211,8 @@ if(UNIT_TEST)
 endif()
 
 
-# install the conversion tool, .so, and all the header files
-INSTALL ( PROGRAMS ${CMAKE_BINARY_DIR}/yuvconvert			DESTINATION bin )
-INSTALL ( TARGETS ${ly_lib_shared} LIBRARY				DESTINATION lib RUNTIME DESTINATION bin )
+# install .so, and all the header files
+INSTALL ( TARGETS ${ly_lib_shared} LIBRARY				DESTINATION lib )
 INSTALL ( DIRECTORY ${PROJECT_SOURCE_DIR}/include/		DESTINATION include )
 
 # create the .deb and .rpm packages using cpack
