diff -up libyuv-0/CMakeLists.txt.1~ libyuv-0/CMakeLists.txt
--- libyuv-0/CMakeLists.txt.1~	2023-01-04 00:24:01.233503908 +0100
+++ libyuv-0/CMakeLists.txt	2023-01-04 00:24:53.998884241 +0100
@@ -36,6 +36,7 @@ SET_TARGET_PROPERTIES	( ${ly_lib_shared}
 if(WIN32)
   SET_TARGET_PROPERTIES	( ${ly_lib_shared} PROPERTIES IMPORT_PREFIX "lib" )
 endif()
+SET_TARGET_PROPERTIES	( ${ly_lib_shared} PROPERTIES VERSION "0.0.0" SOVERSION 0 )
 
 # this creates the conversion tool
 ADD_EXECUTABLE			( yuvconvert ${ly_base_dir}/util/yuvconvert.cc )
